charts/xchemlab/charts/crystal-library/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: compound-library
+description: A service for crystals currently available in the XChem lab
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.1
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.0.5"
```

Comment:
These all need bumped with your latest release, can be done as a separate commit / PR though

Reply:
I've changed them in a separate commit.